### DISCUSSION TITLE
Panopticon won't let me log in (second time)

### DISCUSSION THREAD

**Discussion by morphinestyle** [akeeba/panopticon#533] [Q&A (Community Support)]
This is the second time it's happened to me: Panopticon doesn't want to let me join the administration.

The first time I thought that maybe I had inadvertently changed the password in my password manager (which has never happened to me anyway).
So I reinstalled Panopticon from scratch.

Now it happens again: what could have happened?
(I deleted all cookies but it doesn't solve the problem)

Thank you.

### UPDATE
This time I also tried from mobile and it lets me log in...
I also tried to log in with Firefox Desktop, never used to access Panopticon, but it won't let me log in.
What could cause Chrome and Firefox Desktop to not sign in?

### UPDATE 2
Using Microsoft Edge Desktop (never used to access Panopticon) lets me log in normally...

What could be the problem with Chrome and Firefox Desktop?

**Comment by nikosdion** [comment]
This sounds like some really weird cookie issue I have not seen before. Remember, while Panopticon is new, its underlying framework (AWF) is not; it's been around for almost 11 years. If there was a session bug I would assume that just by the sheer amount of installations we'd have seen it already.

Try removing all cookies from your browser for the domain / subdomain you have installed Panopticon. Does that help?

**Reply by morphinestyle** [reply]
### UPDATE 3

I deleted all the cookies again (which were still empty anyway).

But nothing.

I closed the session by completely closing each instance of Chrome, and restarting it.

But nothing.

I launched Panopticon as anonymous Chrome session: **IT WORKS**.
(I mean: he let me log in by letting me go into the control panel)

**Reply by nikosdion** [reply]
The only thing Incognito does is have a separate cookie and browser storage pool for each tab, which is then cleared once you close that tab. Therefore, you _do_ have a cookie or browser storage problem. You missed something when clearing the cookies. I don't know what.

**Reply by morphinestyle** [reply]
Strange enough: after a while of use in incognito mode it goes back to working correctly even in standard mode.
